includes/core-changes/aspnetcore/5.0/middleware-database-error-page-obsolete.md

```diff
@@ -57,6 +57,8 @@ Complete the following steps:
     }
     ```
 
+1. Add the `Microsoft.AspNetCore.Diagnostics.EntityFrameworkCore` NuGet package to the project file.
```

Comment:
```suggestion
1. Add the [Microsoft.AspNetCore.Diagnostics.EntityFrameworkCore](https://www.nuget.org/packages/Microsoft.AspNetCore.Diagnostics.EntityFrameworkCore) NuGet package to the project file.
```

Reply:
This step should appear before the previous step. The `AddDatabaseDeveloperPageExceptionFilter` method lives in this NuGet package.